app/page.tsx

```diff
@@ -20,17 +22,24 @@ const Home = () => {
         </Box>
       </Box>
       <Box
-        w={{ base: "base", xl: "xl" }}
-        h={{ base: "166px", md: "213px", xl: "261px" }}
+        w={{ base: "base", md: "741px", xl: "xl" }}
+        h={{ base: "323px", md: "411px", xl: "462px" }}
         p={{
-          base: "23px 23px 27px 23px",
+          base: "23px 24px 27px 24px",
           md: "37px 27px 28px 26px",
-          xl: "50px 127px 40px 127px",
+          xl: "50px 128px 40px 127px",
```

Comment:
padding numbers may not exactly match spec, but could be made moot since we may replace map layout with an edge-to-edge one (as per @mjjsf)

Reply:
Moved this to a new issue #97